sql/2020/07_SEO/10_04a_has_hreflang_raw.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+# 10_04a: has hreflang
+SELECT
+  client,
+  COUNTIF(has_hreflang) AS freq,
+  COUNT(0) AS total,
+  ROUND(COUNTIF(has_hreflang) * 100 / COUNT(0), 2) AS pct
+FROM (
+  SELECT
+    client,
+    REGEXP_CONTAINS(body, '(?i)<link[^>]*hreflang') AS has_hreflang
```

Comment:
Proposed custom metric:

```js
Array.from(document.querySelectorAll('link[hreflang]')).map(e => e.getAttribute('hreflang'))
```

Reply:
Like your style. I converted a few to use map.

Reply:
in this case we are already capturing link tags, so I could search that data for the hreflang ones.